src/aspire/basis/basis_utils.py

```diff
@@ -16,23 +16,48 @@
 
 
 def check_besselj_zeros(nu, z):
+    """
+    Verify a sequence of estimated zeros of the Bessel function with order `nu`.
```

Comment:
Verify in what sense?

Reply:
Yeah it doesn't really directly verify in the sense of computing `jv(z)`. It checks that zeros are positive, real, spaced greater than 3 apart, and a few other properties. Do you think I should change this word, or add some kind of additional check like this?